kai/server.py

```diff
@@ -284,6 +288,15 @@ def app():
     KAI_LOG.info(f"Selected provider: {config.models.provider}")
     KAI_LOG.info(f"Selected model: {webapp['model_provider'].model_id}")
 
+    webapp["jinja_env"] = Environment(
```

Comment:
I think I'd put this farther down in the stack. It's more of an implementation detail within the llm_io_handler than something the server needs to be concerned about IMO

Reply:
Sounds good, implementing